CefSharp/AcceleratedPaintInfo.cs

```diff
@@ -2,12 +2,17 @@
 //
 // Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
 
+using System;
+using CefSharp.Enums;
+
 namespace CefSharp
 {
     /// <summary>
     /// AcceleratedPaintInfo
```

Comment:
Please update to match the `CEF API`.

```
Class containing shared texture information for the OnAcceleratedPaint
callback. Resources will be released to the underlying pool for reuse when
the callback returns from client code.
```

Reply:
The underlying CEF code takes care of the resource management.
The callback only passes a handle to the resource.
When using this callback, you just need to know that you cannot use the handle after returning. That's what this documentation says.
As I already said above: make a copy of the resource inside the callback, if you intend to use it outside of the callback.

Reply:
Please add the relevant `xml doc` and change the public fields (the project doesn't use public fields unless they're static) to properties set via the constructor.

Thanks!